lib/src/repository/git_local_repository.dart

```diff
@@ -110,15 +114,22 @@ class GitLocalRepository {
   /// Return the String content of the file in [branch] and [path].
   ///
   /// Throws [GitToolException] if the git command fails.
```

Comment:
What does it do if `maxOutputBytes` is exceeded, throw what exception or truncate the result.

Reply:
Right now `runProc` truncates the results, prepends the `stderr` with `Output exceeded $maxOutputBytes bytes.` and also sets the `wasOutputExceeded` flag. As a result, the `git` operation will fail with a `GitToolException`.